src/api/spec/models/kiwi/package_group_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Kiwi::PackageGroup, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { is_expected.to define_enum_for(:kiwi_type).with(%i[bootstrap delete docker image iso lxc oem pxe split testsuite vmx]) }
```

Comment:
Personally I dont like these kind of expectations, its like all they are is a carbon copy of the original code we are testing. I think testing should be about testing behaviour not just that this has been written.

For example, you wouldn't write a test like this would you?

```ruby
class MyClass
  self.my_var = 'hello_world'
end
```

```ruby
let(:klass) { MyClass.new }

it { expect(klass.my_var).to eq('helloworld') }
```

Thats effectively all that `define_enum_for` and other mathers like `validate_presence_of` do..